torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1098,19 +1098,38 @@ def _make_user_magic(method, user_type):
     else:
         method_attr = method
 
+    def update_ranges(symbol, method, other):
+        min_ = -sympy.oo
+        max_ = sympy.oo
+        if method == 'gt':
+            min_ = other.hint + 1
```

Comment:
Can't base this off hints. The hint could vary on subsequent run but ranges must be valid for ALL runs, since we eliminate guards with them.

Reply:
Yeah this PR isn't it at all, not even worth your review time, I just made this to verify a few assumptions :)